src/tests/asyncTests.ts

```diff
@@ -288,6 +289,12 @@ export class AsyncTests {
       const { apexTestClassIdSet, testResults, globalTests } =
         await this.buildAsyncTestResults(apexTestResults);
 
+      const regularTests = testResults.filter(
```

Comment:
@CristiCanizales I would like to leave the results written to the file as is, TestResultRaw. The file is for debugging.